fingerprintauthhelper/src/main/kotlin/com/pro100svitlo/fingerprintAuthHelper/FahManager.kt

```diff
@@ -280,16 +263,20 @@ internal class FahManager(@NonNull c: Context, l: FahListener?, keyName: String,
                 }
             }
 
+            val keyStore: KeyStore
```

Comment:
Great! :)

I hope some day, I'll find a better way to do such things, but I'm kinda still learning Kotlin. ;)